### PR TITLE
Add condition group for sequence flow

### LINKED ISSUE #52: Conditional props - sequence flow
This should implement Condition section for Conditional Sequence Flow.

### PR DESCRIPTION
**Pull request by barmac**
- chore: move ElementUtil to src/utils
- feat(platform): add condition group

Closes #52

### CODE REVIEW

File: src/provider/camunda-platform/properties/ConditionProps.js [hunk 1]
```diff
@@ -0,0 +1,415 @@
...
+      { value: 'script', label: translate('Inline Script') }
+    ];
+
+    return options;
+  };
+
+  return Select({
+    element,
+    id: 'conditionScriptType',
+    label: translate('Type'),
```
Comment (pinussilvestrus) [inline src/provider/camunda-platform/properties/ConditionProps.js:301]: We have now 2 times an entry calling "Type" in the same group. Maybe we can name this one "Script Type"?

image

File: src/provider/camunda-platform/properties/ConditionProps.js [hunk 2]
```diff
@@ -0,0 +1,415 @@
...
+  const commandStack = useService('commandStack');
+  const translate = useService('translate');
+  const debounce = useService('debounceInput');
+
+  const getValue = () => {
+    return getConditionExpression(element).get('language');
+  };
+
+  const setValue = (value) => {
+    commandStack.execute('properties-panel.update-businessobject', {
```
Comment (pinussilvestrus) [inline src/provider/camunda-platform/properties/ConditionProps.js:242]: Missing test coverage indicated here from CodeCov. 🤔

File: src/provider/camunda-platform/CamundaPlatformPropertiesProvider.js [hunk 3]
```diff
@@ -232,13 +233,18 @@ function ErrorGroup(element) {
   return null;
 }
 
-// @TODO: implement, hide with no entries in the meantime
-function ConditionalGroup(element) {
+/**
+ * Defines condition properties for conditional sequence flow.
+ * Cf. https://docs.camunda.org/manual/latest/reference/bpmn20/gateways/sequence-flow/
+ */
```
Comment (pinussilvestrus) [inline src/provider/camunda-platform/CamundaPlatformPropertiesProvider.js:239]: Maybe we can add this comment to the `ConditionProps`? Just to be consistent with [other props](https://github.com/bpmn-io/bpmn-properties-panel/blob/main/src/provider/camunda-platform/properties/UserAssignmentProps.js#L13).

File: test/spec/provider/camunda-platform/ConditionProps.spec.js [hunk 4]
```diff
@@ -0,0 +1,490 @@
...
+
+        // then
+        expect(input.value).to.eql(originalValue);
+      })
+    );
+  });
+});
+
+
+// helper //////////////////
```
Comment (pinussilvestrus) [inline test/spec/provider/camunda-platform/ConditionProps.spec.js:470]:
```suggestion
// helper //////////////////

```

**Review by pinussilvestrus — COMMENTED** [review]
Great work 👍 just some minimal things to consider

File: src/provider/camunda-platform/properties/ConditionProps.js (hunk 1 above)
Comment (barmac) [reply src/provider/camunda-platform/properties/ConditionProps.js:301]: I'll do that.

File: src/provider/camunda-platform/CamundaPlatformPropertiesProvider.js (hunk 3 above)
Comment (barmac) [reply src/provider/camunda-platform/CamundaPlatformPropertiesProvider.js:239]: OK

**Review by pinussilvestrus — APPROVED** [review]
LGTM 👍

**Comment by barmac** [comment]
@andreasgeier 
image
image

**Review by andreasgeier — APPROVED** [review]
✅ &nbsp;LGTM

**Pull request merged**